### PR TITLE
EditPostActivity - Update showVideoDurationLimitWarning to use Snackbar

### PR DESCRIPTION
**Pull request by geriux**
Currently, there's an issue where the Video length limitation message is getting cut off when it is shown inside a `Toast`:

<kbd></kbd>

The current project's target API is `33` and according to the Toast docs, starting from Android 12 (API 31) it will be limited to two lines. 

> Starting with Android 12 (API level 31), apps targeting Android 12 or newer will have their toasts limited to two lines.
> [Source](https://developer.android.com/reference/android/widget/Toast)

So this PR moves this message to a Snackbar and it will be shown like this:
<kbd></kbd>

-----

## To Test:

- Install the build from this PR
- Open the Editor
- Add the VideoPress block
- Try to add a video that's over the 5 minute limit
- **Expect** to see the Snackbar with the message showing without getting cut off

-----

## Regression Notes

1. Potential unintended areas of impact

    - None, it should only affect this functionality.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - It does not apply for this changes.

3. What automated tests I added (or what prevented me from doing so)

    - Just performed manual testing.

-----

## PR Submission Checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

-----

## Testing Checklist:

- [ ] WordPress.com sites and self-hosted Jetpack sites.
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] Talkback.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] Large and small screen sizes. (Tablet and smaller phones)
- [ ] Multi-tasking: Split screen and Pop-up view. (Android 10 or higher)

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java [hunk 1]
```diff
@@ -3932,7 +3932,12 @@ public void onMediaModelsCreatedFromOptimizedUris(@NonNull Map<Uri, ? extends Me
     }
 
     @Override public void showVideoDurationLimitWarning(@NonNull String fileName) {
-        ToastUtils.showToast(this, R.string.error_media_video_duration_exceeds_limit, ToastUtils.Duration.LONG);
+        String message = getString(R.string.error_media_video_duration_exceeds_limit);
+        WPSnackbar.make(
+                findViewById(R.id.editor_activity),
+                message,
+                Snackbar.LENGTH_LONG
```
Comment (fluiddot) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java:3939]: Nitpick: I found the duration of the snack bar notification a bit short to read the error message. However, I noticed we use the widely parameter `LENGTH_LONG` across the Android app code base, so maybe it's a standard time to wait.

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 ! Thanks @geriux for fixing this 🏅 !

File: WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java (hunk 1 above)
Comment (geriux) [reply WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java:3939]: Yeah, sometimes it is too quick even when setting the `LONG` option. I suspect if users missed the message, they could see it again if they try to upload the same video again.

**Pull request merged**